src/crud.py

```
@@ -0,0 +1,31 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import select
+
+from database import UserModel, UserGroupEnum, UserGroupModel
+from schemas.accounts import CreateUser
+
+from security.passwords import hash_password
+
+
+async def create_user(db: AsyncSession, user: CreateUser):
+    hashed = hash_password(user.password)
+    result = await db.execute(
+        select(UserGroupModel).where(
+            UserGroupModel.name == UserGroupEnum.USER.value
+        )
+    )
+    group = result.scalar_one()
+    db_user = UserModel(
+        email=user.email, _hashed_password=hashed, group_id=group.id
+    )
+    db.add(db_user)
+    await db.commit()
+    await db.refresh(db_user)
```

The function commits and refreshes the created user which is correct. One small robustness improvement: if the UserGroup row is somehow missing, `result.scalar_one()` will raise; you may want to catch that (or let it bubble if seeding guarantees groups exist). Tests typically seed groups, so this is not strictly required, but consider whether you want a clearer error in that case.